src/components/SlateEditor/PlainTextEditor.tsx

```diff
@@ -38,6 +39,18 @@ const PlainTextEditor = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const editor = useMemo(() => withHistory(withReact(withPlugins(createEditor(), plugins))), []);
 
+  const formikContext = useFormikContext<LearningResourceFormikType>();
+  const { status, setStatus } = formikContext;
```

Comment:
Kunne denne vore definert slik eller må du ha formikContext som objekt?
```suggestion
  const { status, setStatus } = useFormikContext<LearningResourceFormikType>();
```